java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1975,6 +1976,21 @@ public T type(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Renames <code>key</code> to <code>newKey</code> if <code>newKey</code> does not yet exist.
```

Comment:
We don't add to transaction, because
1) transaction could be routed to one node only
2) if route not given, GLIDE core picks a route for the transaction according to the keys used referenced in it

A user may still route a transaction to a wrong node intentionally, on it own responsibility.